internal/generator/gen_test.go

```diff
@@ -4,19 +4,21 @@ import (
 	"io"
 	"log"
 	"os"
-	"path"
 	"path/filepath"
 	"strings"
 	"testing"
 
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/woven-planet/go-zserio/internal/ast"
 	"github.com/woven-planet/go-zserio/internal/model"
 )
 
-func testWorkspace(filePath string) string {
-	return path.Join(os.Getenv("TEST_SRCDIR"), os.Getenv("TEST_WORKSPACE"), filePath)
+func testWorkspace(filePath string, t require.TestingT) string {
```

Comment:
nit: `testing.T` might be nice to have as a first argument and depending on `*testing.T` might be OK here.

Reply:
`*testing.T` doesn't work with benchmarks, so I chose `require.TestingT`, which works for both tests and benchmarks.